packages/babel-register/loader.mjs

```diff
@@ -0,0 +1,21 @@
+/*import {
+  load as _load,
+  resolve as _resolve,
+  getSource as _getSource,
+  getFormat as _getFormat,
+  globalPreload as _globalPreload,
+  getGlobalPreloadCode as _getGlobalPreloadCode,
+} from "./lib/loader.mjs";
+
+const [major] = process.versions.node.split(".").map(Number);
+
+export const load = major >= 16 && _load;
+export const resolve = major >= 16 && _resolve;
+export const globalPreload = major >= 16 && _globalPreload;
+
+export const getSource = major < 16 && _getSource;
```

Comment:
Since this is all experimental anyway, I would prefer to only support Node.js 16&18 and simplify our implementation.

Reply:
The backwards compatibility - at least down to node 14 - consists of renamed functions only plus getFormat, which is 5 lines.